libbeat/beat/pipeline.go

```diff
@@ -21,11 +21,13 @@ import (
 	"time"
 
 	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/outputs"
 )
 
 type Pipeline interface {
 	PipelineConnector
 	SetACKHandler(PipelineACKHandler) error
+	GetOutputGroup() outputs.Group // FIXME! Causes circular import
```

Comment:
I'd like some guidance on how to fix this (again, assuming this PR is heading in the right direction, of course).